.github/workflows/e2e_custom_cl.yml

```diff
@@ -48,78 +33,29 @@ jobs:
       AWS_REGION: ${{ secrets.QA_AWS_REGION }}
       AWS_ROLE_TO_ASSUME: ${{ secrets.QA_AWS_ROLE_TO_ASSUME }}
       QA_KUBECONFIG: ${{ secrets.QA_KUBECONFIG }}
+      QA_PRIVATE_GHA_PULL: "false"
   e2e_cutsom_run_smoke_tests:
     name: E2E Custom Run Smoke Tests
     runs-on: ubuntu-latest
     needs: [e2e_custom_build_artifacts, e2e_custom_build_custom_chainlink_image]
     env:
       CGO_ENABLED: 1
     steps:
-      - name: Checkout the repo
-        uses: actions/checkout@f25a3a9f25bd5f4c5d77189cab02ff357b5aedeb # v2.4.1
-      - name: Install Nix
```

Comment:
So all of these steps are now removed and there are implicitly run where?

This is now replaced with the `smartcontractkit/ctf-ci-e2e-action@v1.0.0` action which I assume takes care of some of these steps, but what about this repo custom steps like preparing the NIX env and running "Generate gauntlet executable", where is that run with the new flow?